src/main/java/org/mitre/synthea/modules/LifecycleModule.java

```diff
@@ -714,14 +714,19 @@ private static double reverseCalculateCreatinine(Person person, double crcl, lon
     }
   }
 
+  private static final boolean ENABLE_DEATH_BY_NATURAL_CAUSES =
+      Boolean.parseBoolean(Config.get("lifecycle.death_by_natural_causes"));
+  
   private static final Code NATURAL_CAUSES = new Code("SNOMED-CT", "9855000",
       "Natural death with unknown cause");
 
   private static void death(Person person, long time) {
-    double roll = person.rand();
-    double likelihoodOfDeath = likelihoodOfDeath(person.ageInYears(time));
-    if (roll < likelihoodOfDeath) {
-      person.recordDeath(time, NATURAL_CAUSES, "death");
+    if (ENABLE_DEATH_BY_NATURAL_CAUSES) {
```

Comment:
Calling this out to get more attention to it -- I added this because for a veteran population, the target ages were on average higher than the general population, and so to generate a large set of living patients there were a lot of dead patients which added no value and just slowed down the process. Even in a normal population, most use cases only care about living patients, so adding more dead ones which only have a sudden and completely random "natural death by unknown causes" doesn't add value, and again it slows things down and takes up disk space.

My preference is to leave the config setting disabled by default, and if people specifically want dead patients they can enable it.

Reply:
Agreed. The setting is disabled by default in the updates I've made.